contract/src/msgs/data_requests/test_helpers.rs

```diff
@@ -285,11 +285,15 @@ impl TestAccount {
     pub fn get_data_requests_by_status(
         &self,
         status: DataRequestStatus,
-        offset: u32,
+        last_seen_index: Option<(Uint128, u64, Hash)>,
```

Comment:
Should we use the `pub type IndexKey`?

Reply:
The types here are actually different. `Uint128` vs `u128` for json purposes its the former, the latter is for sorting correctly.

Reply:
Maybe this can be a type in common? Since we repeat it a couple times in there as well.

Reply:
What should we name it to differentiate it? `MessageIndexKey`?

Reply:
Maybe even more bespoke? `LastSeenIndexKey`? :P